AzFilesHybrid/AzFilesHybrid.psd1

```diff
@@ -97,6 +97,7 @@
         "Join-AzStorageAccount", 
         "Invoke-AzStorageAccountADObjectPasswordRotation",
         "Debug-AzStorageAccountAuth",
+        "Update-AzStorageAccountSetupForAES256",

```

Comment:
Nit: Include auth in name? Something like Update-AzStorageAccountAuthSetupForAES256 or Update-AzStorageAccountAuthForAES256